src/xrEngine/xr_ioc_cmd.cpp

```diff
@@ -713,6 +728,7 @@ void CCC_Register()
 
     // Render device states
     CMD4(CCC_Integer, "r__supersample", &ps_r__Supersample, 1, 4);
+    CMD2(CCC_SunDirection, "r__sun_direction", &ps_sun_direction);
```

Comment:
rs_sun_direction seems more outlined with such commands as rs_stats, rs_fullscreen, rs_gamma and others than r__sun_direction with r__ rendering commands. Maybe the placement isn't good, but name is pretty matches to the other commands